examples/federated/index.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line
+import('./bootstrap');
```

Comment:
No longer required, if using import()

Reply:
It looks like it will be required again pending https://github.com/webpack/webpack/pull/10960 based on Tobias' comments.